### DISCUSSION TITLE
how to download music with spotiflyer app

### DISCUSSION THREAD

**Discussion by DJklusterfuQ** [Shabinder/SpotiFlyer#865] [Q&A] [answered]
How do you use this app as far as downloading music goes? I'll go into spotify through through the spotiflyer app But I see no way to download the songs or playlist it just kind of mimics spotify. is  Is there a download button? I've searched but cannot seem to find one. Please help.

**Comment by Shabinder** [comment]
share from spotify to this app

**Comment by ElizabethWega** [comment, marked as answer]
> How do you use this app as far as downloading music goes? I'll go into spotify through through the spotiflyer app But I see no way to download the songs or playlist it just kind of mimics spotify. is  Is there a download button? I've searched but cannot seem to find one. Please help.

Click 'share' in Spotify application and press 'copy link'. After that enter the SpotiFlyer app and paste it into the search bar. The app will fetch the song(s) for you. At the bottom you will see the 'download all' button if you want to download all of the songs from the album or playlist. You also have a download icon on the right side of every track if you want to download just some of the tracks. Take note that sometimes the download may fail...it takes few tries(for me) to make it fetch and download the track...if it keeps happening, then the app cannot find the corresponding link of the song on JioSaavan. Hope this helps. Cheers!
